extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -143,6 +145,21 @@ CapabilityBuildItem register(
             }
         }
 
+        // register @JsonIdentityInfo strategy implementations for reflection
+        for (AnnotationInstance jsonIdentityInfoInstance : index.getAnnotations(JSON_IDENTITY_INFO)) {
+            AnnotationValue generatorValue = jsonIdentityInfoInstance.value("generator");
+            AnnotationValue resolverValue = jsonIdentityInfoInstance.value("resolver");
+            if (generatorValue != null) {
+                reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, generatorValue.asClass().name().toString()));
+            }
+            if (resolverValue != null) {
+                reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, resolverValue.asClass().name().toString()));
+            } else {
+                // Tried adding the jackson databind dependency to the index so we can use default value; didn't work
```

Comment:
I think we can improve this. Comment. Just mention that this class is the default value of the annotation

Reply:
I understand the reasoning, but putting the explanation in the source doesn't bring much - we would essentially have to do the same thing everywhere we are using the default value of an annotation.

Just add a comment mentioning that it's the default and that is why it's being registered